src/main/java/meteordevelopment/meteorclient/systems/waypoints/Waypoints.java

```diff
@@ -41,9 +43,9 @@
 
     private static final Color TEXT = new Color(255, 255, 255);
 
-    public final Map<String, AbstractTexture> icons = new HashMap<>();
+    public final ConcurrentHashMap<String, AbstractTexture> icons = new ConcurrentHashMap<>();
 
-    public List<Waypoint> waypoints = new ArrayList<>();
+    public ConcurrentHashMap<String, Waypoint> waypoints = new ConcurrentHashMap<>();
```

Comment:
Can you use `Map<String, Waypoint>` as using the base abstract type is the standard. Same with the one above.

Reply:
Done